module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -250,8 +250,7 @@ public function getPreviewLinks()
      */
     public function supportsAjaxStatus()
     {
-        // TODO: add this as an Overdrive configuration to turn it off
-        return true;
+        return $this->config->enableAjaxStatus;
```

Comment:
Should we have a default value here in case enableAjaxStatus is unset? e.g. `return $this->config->enableAjaxStatus ?? true;`?